test/dynamo/test_torchxla_integration.py

```diff
@@ -131,3 +131,8 @@ class TorchXLAReuseGraphTest(unittest.TestCase):
     test_matmul = make_reuse_graph_test(MatmulModule)
     test_linear = make_reuse_graph_test(LinearModule)
     test_inplace_update = make_reuse_graph_test(ModuleInplaceUpdate)
+
+
+if __name__ == "__main__":
+    test = unittest.main()
```

Comment:
I remember I encounter CI failure because of unrecognized arguments before when I added this.

Please keep an eye on the CI results